geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/Put70.java

```diff
@@ -411,6 +413,9 @@ public void cmdExecute(final Message clientMessage, final ServerConnection serve
       serverConnection.setAsTrue(RESPONDED);
       region.getCachePerfStats().incDeltaFullValuesRequested();
       return;
+    } catch (UnavailableSecurityManagerException e) {
```

Comment:
So theoretically every command needs to do this?

Reply:
@jinmeiliao yes, but the issue has only occurred on put70 command so far. 
There are (if I'm not mistaken) 17 other commands that can hit the same issue so I can make the change there too.

Reply:
Is there a generic way to fix this without having to touch all the command files?